<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndNewGame</class>
 <widget class="QWidget" name="wndNewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>164</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>164</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nowa gra</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbGame">
     <property name="title">
      <string>Gra</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lbGameTurn">
        <property name="text">
         <string>Czas tury:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbGameTurn">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>500</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbBoard">
     <property name="title">
      <string>Plansza</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="lbBoardHeight">
        <property name="text">
         <string>Wysokość:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbBoardWidth">
        <property name="text">
         <string>Szerokość:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbBoardWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbBoardHeight">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbPawns">
     <property name="title">
      <string>Pionki</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lbPawnsHunters">
        <property name="text">
         <string>Naganiacze:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbPawnsHunters">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbPawnsAnimals">
        <property name="text">
         <string>Zwierzęta:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbPawnsAnimals">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layButtons">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbGameTurn</tabstop>
  <tabstop>sbPawnsHunters</tabstop>
  <tabstop>sbPawnsAnimals</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>pressed()</signal>
   <receiver>wndNewGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
